### PR TITLE
Closes #1482: Event tile

### LINKED ISSUE #1482: Tile for Event
https://docs.google.com/document/d/1NhIta9XU5Lv1-1OmnTCYTNc-613VHrhzRZ0FqsOTq3M/edit?ts=5be2ab9a

### PR DESCRIPTION
**Pull request by dnarcese**
### Pull Request checklist

- [x] This PR includes thorough **tests** or an explanation of why it does not
Temporary change
- [x] This PR includes a **CHANGELOG entry** or does not need one

### CODE REVIEW

File: app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/BrowserNavigationOverlay.kt [hunk 1]
```diff
@@ -204,9 +205,12 @@ class BrowserNavigationOverlay @JvmOverloads constructor(
 
         val manager = HomeTileManager(context, COL_COUNT)
         manager.spanSizeLookup = object : GridLayoutManager.SpanSizeLookup() {
+
+            val localeIsEnUs = LocaleManager.getInstance().isLocaleENUS(context)
```
Comment (liuche) [inline app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/BrowserNavigationOverlay.kt]: Nit: this is slightly inconsistent with the other localeIsENUS - I don't care which it is, but I think they should be consistent. (But you're right, pretty much all of these are terrible.)

File: app/src/main/java/org/mozilla/tv/firefox/pinnedtile/PinnedTileAdapter.kt [hunk 2]
```diff
@@ -32,6 +33,9 @@ import org.mozilla.tv.firefox.ext.withRoundedCorners
 import org.mozilla.tv.firefox.telemetry.TelemetryIntegration
 import org.mozilla.tv.firefox.utils.FormattedDomain
 
+const val EVENT_TILE = 0
```
Comment (liuche) [inline app/src/main/java/org/mozilla/tv/firefox/pinnedtile/PinnedTileAdapter.kt]: Nit: add a comment?

**Comment by dnarcese** [comment]
@liuche 
# Request for data collection review form

1) What questions will you answer with this data?

- How often do users click the Live Nation concert tile?

2) Why does Mozilla need to answer these questions?  Are there benefits for users? Do we need this information to address product or business requirements? Some example responses:

* Determine whether users are interested and engaged by the Firefox Concert Series.


3) What alternative methods did you consider to answer these questions? Why were they not sufficient?

- This is the only way to know how often the tile is clicked.

4) Can current instrumentation answer these questions?

- No telemetry currently exists to answer these questions.

5) List all proposed measurements and indicate the category of data collection for each measurement, using the Firefox [data collection categories](https://wiki.mozilla.org/Firefox/Data_Collection) on the Mozilla wiki.   

**Note that the data steward reviewing your request will characterize your data collection based on the highest (and most sensitive) category.**

| Measurement Description                                      | Data Collection Category | Tracking Bug #                                               |
| ------------------------------------------------------------ | ------------------------ | ------------------------------------------------------------ |
| When the user clicks the Live Nation concert tile       | 2                        | [Issue 1482](https://github.com/mozilla-mobile/firefox-tv/issues/1482) |

6) How long will this data be collected?  Choose one of the following:

* @athomasmoz will monitor this data as long as the Firefox Concert Series is happening. 

7) What populations will you measure?

* All users in the US, with telemetry enabled, on release channel.


8) If this data collection is default on, what is the opt-out mechanism for users?

- There is an option in settings to stop sending usage data.

9) Please provide a general description of how you will analyze this data.

- The data will be analyzed through Redash queries and dashboards.

10) Where do you intend to share the results of your analysis?

- On Redash and with mobile teams.

**Comment by liuche** [comment]
1) Is there or will there be **documentation** that describes the schema for the ultimate data set available publicly, complete and accurate?
Documented here, for 1-week data collection

2) Is there a control mechanism that allows the user to turn the data collection on and off?
Yes, controlled by telemetry toggle

3) If the request is for permanent data collection, is there someone who will monitor the data over time?**
No, removed in a week

4) Using the **[category system of data types](https://wiki.mozilla.org/Firefox/Data_Collection)** on the Mozilla wiki, what collection type of data do the requested measurements fall under?  **
Type 2, user interaction by clicking on suggested content

5) Is the data collection request for default-on or default-off?
default on

6) Does the instrumentation include the addition of **any *new* identifiers** (whether anonymous or otherwise; e.g., username, random IDs, etc.  See the appendix for more details)?
No, UI interaction only

7) Is the data collection covered by the existing Firefox privacy notice? **If unsure: escalate to legal if:**
Yes, clicks on promotional tile

8) Does there need to be a check-in in the future to determine whether to renew the data? (Yes/No) (If yes, set a todo reminder or file a bug if appropriate)**
No, this will expire in Dec

**Pull request merged**